pageserver/src/tenant/timeline.rs

```diff
@@ -920,6 +932,16 @@ impl Timeline {
                 warn!("failed to await for frozen and flushed uploads: {e:#}");
             }
         }
+
+        // Page request handlers might be waiting for LSN to advance: they do not respect Timeline::cancel
+        // while doing so.
+        self.last_record_lsn.shutdown();
+
+        tracing::info!("Waiting for tasks...");
```

Comment:
```suggestion
        tracing::debug!("Waiting for tasks...");
```

The task shutdown already includes logging in case it takes a longer time, so I don't think this is useful.